### PR TITLE
LG-14879: Update docs for IAL2 [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by nprimak**
Ticket:
https://cm-jira.usa.gov/browse/LG-14879

Description:
Updating the accordions describing the available service levels and the deprecated values in both OIDC and SAML sections of the dev docs

Refer to google doc linked in ticket for specific language

### CODE REVIEW

**Review by ajfarkas — CHANGES_REQUESTED** [review]
You've got two extra div closing tags appearing as text under the Deprecated Service Values accordion
Screenshot 2024-11-01 at 9 54 50 AM

File: _includes/snippets/auth_content/deprecated_values.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{% capture deprecated_values %}
+  These are not recommended, and only for legacy compatibility.
+  - **`http://idmanagement.gov/ns/assurance/ial/1`**
+      Basic identity assurance, does not require identity verification (this is the most common value). Equivalent to `urn:acr.login.gov:auth-only`.
+  - **`http://idmanagement.gov/ns/assurance/ial/2`**
+      Requires that the user has gone through basic identity verification.Equivalent to `urn:acr.login.gov:verified`.
```
Comment (Sgtpluck) [inline _includes/snippets/auth_content/deprecated_values.md:6]:
```suggestion
      Requires that the user has gone through basic identity verification. Equivalent to `urn:acr.login.gov:verified`.
```

File: _pages/saml/authentication.md [hunk 2]
```diff
@@ -72,8 +72,8 @@ A proofed identity request at AAL2, with phishing resistent MFA, for email, phon
                 {% include accordion.html content=service_levels accordion_id="service_level_accordion"  title="Type of Service Level" id="service_level" %}
                 {% include accordion.html content=aal_values accordion_id="aal_accordion" title="Authentication Assurance (AAL) Values" id="aal_values" %}
                 {% include accordion.html content=attributes accordion_id="attributes_accordion" title="Attributes" id="attributes" %}
-                {% include accordion.html content=loa_values accordion_id="loa_accordion" title="Level of Assurance (LOA) Values (Deprecated)" id="loa_values" %}
-            </dl>
+                {% include accordion.html content=deprecated_values accordion_id="deprecated_accordion" title="Deprecated Service Values" id="deprecated_values" %}
+            </div>
```
Comment (Sgtpluck) [inline _pages/saml/authentication.md:76]: it looks like this </dl> turned into a </div>

Comment (nprimak) [reply _pages/saml/authentication.md]: should be fixed now I think

**Comment by ajfarkas** [comment]
Still seeing formatting errors:
• spacing after the accordion is missing
• prompt and redirect_uri are not 100% width
Screenshot 2024-11-01 at 11 45 00 AM

**Comment by ajfarkas** [comment]
Aaaah, I'm sorry: on `saml/authentication` in tha Attributes accordion, the example IAL Authentication Context is the deprecated IAL2 string.
`<saml:AuthnContextClassRef>http://idmanagement.gov/ns/assurance/ial/2</saml:AuthnContextClassRef>`

**Comment by nprimak** [comment]
> Aaaah, I'm sorry: on `saml/authentication` in tha Attributes accordion, the example IAL Authentication Context is the deprecated IAL2 string. `<saml:AuthnContextClassRef>http://idmanagement.gov/ns/assurance/ial/2</saml:AuthnContextClassRef>`

No need to apologize! I'm glad you're catching all this stuff

**Comment by nprimak** [comment]
@Sgtpluck  just want to double check with you since AJ's comment isn't part of the google doc updates - do we want to update that attributes accordion example IAL Authentication Context ?

**Comment by Sgtpluck** [comment]
@nprimak yes, if you could do a search for the old IAL values and replace all of them with the semantic equivalents, that would be great. thanks!

**Comment by nprimak** [comment]
Updated all the deprecated values with their new semantic equivalents

**Comment by Sgtpluck** [comment]
thanks for catching and correcting all those values!

**Comment by ajfarkas** [comment]
:/ the sidebar examples all have deprecated values:   
/saml/authentication/ Example,   
/oidc/authorization/ JWT Request, PKCE Request

**Comment by Sgtpluck** [comment]
yeah it looks like the pkce/jwt values got missed because they are encoded in the example. i am wondering how useful it is to display them that way?

**Comment by nprimak** [comment]
@Sgtpluck  yeah that is why I missed them but because the new values don't have slashes the encoding isn't an issue anymore.  I put it through an HTML encoder and it still looked the same

**Pull request merged**